pandas/io/html.py

```diff
@@ -181,6 +185,12 @@ class _HtmlFrameParser:
     displayed_only : bool
         Whether or not items with "display:none" should be ignored
 
+    extract_links : {None, "all", "header", "body", "footer"}
+        Table elements in the specified section(s) with <a> tags will have their
+        href extracted.
+
+    .. versionadded:: 1.5.0
```

Comment:
```suggestion
        .. versionadded:: 1.5.0
```

Reply:
Ah, thank you very much - I didn't notice that the indentation was off, and the error message was a bit cryptic so I didn't make the connection.